site/source/docs/getting_started/downloads.rst

```diff
@@ -7,10 +7,10 @@ Download and install
 .. note:: You can also :ref:`build Emscripten from source <installing-from-source>` if you prefer that to downloading binaries using the emsdk.
 
 .. note:: There are additional ways to install Emscripten than the instructions
-    below, for example, using brew on MacOS, the package manager on your linux
-    distro, or a Docker image, etc. However, the emsdk is the only officially
-    supported way to use Emscripten that is supported by the Emscripten
-    project, and the only one that we constantly test
+    below, for example, using homebrew on MacOS, the package manager on your linux
```

Comment:
```suggestion
    below, for example, using Homebrew on MacOS, the package manager on your Linux
```

if we use a tool name and not command name, probably need to use official capitalisation.